src/index.js

```diff
@@ -0,0 +1,36 @@
+import './style.css';
+
+const tasks = [
+  {
+    description: 'Wake up',
+    completed: true,
+    index: 0,
+  },
+  {
+    description: 'Pray',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'Eat breakfast',
+    completed: false,
+    index: 2,
+  },
+];
+
+const taskWrapper = document.querySelector('.to-dos');
+
+const showList = () => {
+  taskWrapper.innerHTML = '';
+  for (let i = 0; i < tasks.length; i += 1) {
+    taskWrapper.innerHTML += `
+    <form id="form">
+      <input type="checkbox" id="${tasks[i].index}" name="task" value="task">
+      <label for="${tasks[i].index}">${tasks[i].description}</label>
+      <i class="fa fa fa-times"></i><br>
+    </form>
```

Comment:
- [ ] It would be better semantically if you add every task as an unordered list item inside a `<li>` tag because it's not right to have the list of tasks as multiple forms like you do here, So please fix that.

![image](https://user-images.githubusercontent.com/68796232/167835053-7256a9eb-1079-4af8-ba38-d661e7486e5c.png)